packages/rx_bloc_cli/mason_templates/bricks/lib_pin_code/__brick__/lib/lib_pin_code/views/verify_pin_code_page.dart

```diff
@@ -0,0 +1,142 @@
+{{> licence.dart }}
+
+import 'package:flutter/material.dart';
+import 'package:provider/provider.dart';
+import 'package:widget_toolkit/models.dart';
+import 'package:widget_toolkit/ui_components.dart';
+import 'package:widget_toolkit_biometrics/widget_toolkit_biometrics.dart';
+import 'package:widget_toolkit_pin/widget_toolkit_pin.dart';
+
+import '../../base/common_blocs/coordinator_bloc.dart';
+import '../../l10n/l10n.dart';
+import '../bloc/pin_bloc.dart';
+import '../models/pin_code_arguments.dart';
+
+class VerifyPinCodePage extends StatefulWidget {
+  const VerifyPinCodePage({
+    this.pinCodeArguments = const PinCodeArguments(
+      title: '',
+      isSessionTimeout: false,
+    ),
+    super.key,
+  });
+
+  final PinCodeArguments pinCodeArguments;
+
+  @override
+  State<VerifyPinCodePage> createState() => _VerifyPinCodePageState();
+}
+
+class _VerifyPinCodePageState extends State<VerifyPinCodePage> {
+  @override
+  void initState() {
+    WidgetsBinding.instance.addPostFrameCallback((_) {
+      context.read<PinBlocType>().events
+        ..setPinCodeType(widget.pinCodeArguments.isSessionTimeout)
+        ..temporaryDisableBiometrics(false);
+    });
+    super.initState();
+  }
+
+  @override
+  Widget build(BuildContext context) => Builder(
+        builder: (context) => WillPopScope(
+          onWillPop: () async {
+            context.read<PinBlocType>().events.deleteSavedData();
+            return true;
+          },
+          child: Scaffold(
+            appBar: AppBar(
+              title: Text(
+                widget.pinCodeArguments.title.isEmpty
+                    ? context.l10n.libPinCode.verifyPinCodePage
+                    : widget.pinCodeArguments.title,
+              ),
+            ),
+            extendBodyBehindAppBar: true,
+            body: SizedBox(
+              height: MediaQuery.of(context).size.height,
```

Comment:
Here it's better to use MediaQuery.sizeOf(context).height to avoid unnecessary rebuilds

Reply:
this was changed to `MediaQuery.sizeOf(context).height`